nvflare/apis/responder.py

```diff
@@ -76,6 +76,18 @@ def process_task_check(self, task_id: str, fl_ctx: FLContext):
         """
         pass
 
+    @abstractmethod
+    def process_job_heartbeat(self, fl_ctx: FLContext):
```

Comment:
We call this method not just when processing job heartbeats though.
Or should we say something like ``handle_client_interaction`` or ``process_client_interaction`` or ``process_client_update`` or ``handle_client_update``?

Reply:
We can do these in 2.5. Let's not spend time polishing this for 2.3.
In fact, we need to redesign this in 2.5. The logic for determining dead client should be moved to server_runner.